### DISCUSSION TITLE
run the ROCKET model，there is a error. TypeError: __init__() got an unexpected keyword argument 'comment'

### DISCUSSION THREAD

**Discussion by hellozhq** [timeseriesAI/tsai#437] [Q&A] [answered]
run the code beside:
**from tsai.all import *
X, y, splits = get_UCR_data('HandMovementDirection', return_split=False)
tfms  = [None, [Categorize()]]
batch_tfms = [TSStandardize(by_sample=True)]
dsets = TSDatasets(X, y, tfms=tfms, splits=splits)
dls = TSDataLoaders.from_dsets(dsets.train, dsets.valid, bs=64, drop_last=False, shuffle_train=False, batch_tfms=[TSStandardize(by_sample=True)])
model = create_model(ROCKET, dls=dls)
X_train, y_train = create_rocket_features(dls.train, model)
X_valid, y_valid = create_rocket_features(dls.valid, model)
ridge = RidgeClassifierCV(alphas=np.logspace(-8, 8, 17), normalize=True)
ridge.fit(X_train, y_train)
print(f'alpha: {ridge.alpha_:.2E}  train: {ridge.score(X_train, y_train):.5f}  valid: {ridge.score(X_valid, y_valid):.5f}')**

Traceback (most recent call last):
  File "D:/JetBrains/timeseries/test.py", line 29, in <module>
    X_train, y_train = create_rocket_features(dls.train, model)
  File "D:\JetBrains\timeseries\venv\lib\site-packages\tsai\models\ROCKET.py", line 140, in create_rocket_features
    for i,(xb,yb) in enumerate(progress_bar(dl, display=verbose, leave=False, comment='batch/batches')):
TypeError: __init__() got an unexpected keyword argument 'comment'

Process finished with exit code 1

I just run in the cpu, my computer is AMD Computer processor。

**Comment by oguiza** [comment]
Hi @hellozhq,
I think it may be an issue with the version you are using. Could you please run
```python
from tsai.all import *
import fastprogress
my_setup(fastprogress)
```
in your notebook and post your results?
I've run this code:
```python
from tsai.all import *
from sklearn.linear_model import RidgeClassifierCV
X, y, splits = get_UCR_data('HandMovementDirection', return_split=False)
tfms = [None, [Categorize()]]
batch_tfms = [TSStandardize(by_sample=True)]
dsets = TSDatasets(X, y, tfms=tfms, splits=splits)
dls = TSDataLoaders.from_dsets(dsets.train, dsets.valid, bs=64, drop_last=False, shuffle_train=False, batch_tfms=[TSStandardize(by_sample=True)])
model = create_model(ROCKET, dls=dls)
X_train, y_train = create_rocket_features(dls.train, model)
X_valid, y_valid = create_rocket_features(dls.valid, model)
ridge = RidgeClassifierCV(alphas=np.logspace(-8, 8, 17), normalize=True)
ridge.fit(X_train, y_train)
print(f'alpha: {ridge.alpha_:.2E} train: {ridge.score(X_train, y_train):.5f} valid: {ridge.score(X_valid, y_valid):.5f}')
```
using this setup: 
```bash 
os             : Linux-5.4.144+-x86_64-with-Ubuntu-18.04-bionic
python         : 3.7.12
tsai           : 0.3.1
fastai         : 2.5.3
fastcore       : 1.3.29
fastprogress   : 1.0.2
torch          : 1.10.0+cu111
device         : 1 cuda (['Tesla P100-PCIE-16GB'])
n_cpus         : 2
RAM            : 12.69 GB
GPU memory     : [15.9] GB
```
and it works well.
You may need to upgrade fastprogress using:
```python
pip install fastprogress
```
or
```python
conda install -c fastai fastprogress
```

**Reply by z-979** [reply]
hi，@oguiza，I have kept the version consistent with you, but there are still the same problems
```
os             : Windows-10-10.0.19041-SP0
python         : 3.7.13
tsai           : 0.3.1
fastai         : 2.5.3
fastcore       : 1.3.29
fastprogress   : 1.0.2
torch          : 1.10.2+cpu
device         : cpu
cpu cores      : 8
RAM            : 15.79 GB
GPU memory     : [2.0] GB，
```

**Comment by z-979** [comment]
Have you solved the problem now？

**Comment by z-979** [comment, marked as answer]
I have solved this problem. I have reinstalled a new environment. The setup is as follows. It can run.
Maybe it's the problem of **cpu** and **gpu**

```
os              : Windows-10-10.0.19044-SP0
python          : 3.8.13
tsai            : 0.3.4
fastai          : 2.7.10
fastcore        : 1.5.27
fastprogress    : 1.0.3
torch           : 1.8.1
device          : 1 gpu (['NVIDIA GeForce MX450'])
cpu cores       : 4
threads per cpu : 2
RAM             : 15.79 GB
GPU memory      : [2.0] GB
```
